api/models.py

```diff
@@ -14,3 +14,22 @@ def __str__(self):
         """one-line docstring for representing the Profile object."""
         return f'{self.user.first_name}'
 
+
+
+
+
+class add_loan_record(models.Model):
+    loan=models.CharField(max_length=200, null=True)
+    user_id=models.ForeignKey(User,on_delete=models.CASCADE)
+    due_date=models.DateField(auto_now=True)
+    created=models.DateField(auto_now_add=True)
+    amount=models.TextField()
+    interest_rate=models.IntegerField()
```

Comment:
The Interest field should accept blank and data type should be characters.

Reply:
the interest rate would be in number(figure) an that is why is in integer....... I would change it to DecimalField because interest rate can be in decimal fiqure